package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "designsystem",
-  "version": "0.8.30",
+  "version": "0.8.31",
```

Comment:
Is this a mistake? Kirby should bump it's version by the build pipeline, so unless there's a special reason this should be reverted.

Reply:
Yes, it's a mistake... I forgot that it automatically performed "the bump"... reverting my change!